tests/test_simple.py

```diff
@@ -315,6 +315,19 @@ def test_single_image():
             'type': 'Rho',
             'file_name': rho_psf_file
         },
+        {
+            'type': 'TwoDHist',
+            'file_name': os.path.join('output', 'simple_psf_twodhiststats.png'),
```

Comment:
In my portability tests recently, I found that .png files had issues on one or two systems.  Something about the libpng library.  I don't know whether it is a bug in matplotlib or with the system's libpng library or what exactly, but using .pdf files (with the 'Agg' backend) worked everywhere, so I think we should stick with that for tests such as these.

If people have a working png parser on their system, obviously that would still work for them.  But I don't want to have my unit tests fail because of some png problem with matplotlib.